x-pack/plugins/security_solution/common/test/ess_roles.json

```diff
@@ -0,0 +1,136 @@
+{
```

Comment:
cc @pheyos looks related to https://github.com/elastic/kibana/issues/169952. If we lack a single source of truth for these roles, we might quickly end up with outdated role definitions here and there, which could make our tests less accurate.

Reply:
This file `ess_roles.json` contains security ESS roles. This PR does also some clean up to reuse this file.

Contrary `packages/kbn-es/src/serverless_resources/security_roles.json` contains Serverless Security roles and definitely requires some automation approach.